.github/workflows/test_with_doctrine.yaml

```diff
@@ -31,4 +31,7 @@ jobs:
                     # do not intall doctrine/orm phpstan, it conflicts with Retor's one
                     composer install -d orm --no-dev
 
+                    # remove phpstan/phpstan to prevent autolaod conflicts
+                    composer remove phpstan/phpstan -d orm
```

Comment:
Since phpstan is phar-ed and prefixed, can it still cause autoload conflicts?

Reply:
Yes, like this:
https://github.com/rectorphp/rector/pull/2800/checks?check_run_id=430026084